README.md

```diff
@@ -280,17 +292,17 @@ for any specific image version before pulling Jenkins image.
 
 ```bash
 # Launch Jenkins container and automatically pull the image if not present
-docker run -d -p 9090:8080 --name=jenkins-master jenkins/jenkins
+docker run -d -p 9090:8080 --name=jenkins-host jenkins/jenkins
```

Comment:
nit: Host does not really add anything here. Jenkins allows for `master` and `agent`(s). In that case it is important to differentiate. Here you might as well just name container jenkins.